my_game.py

```diff
@@ -329,6 +329,15 @@ def on_update(self, delta_time):
         # Update the player shots
         self.player_shot_list.update()
 
+        # Time between asteroid spawn count down
+        self.asteroids_timer -= 1 * delta_time
```

Comment:
What do you get from multiplying by one?

Reply:
What i'm doing should be very clear to everyone.

but I can fix fix